lib/Worker.js

```diff
@@ -195,7 +195,7 @@ module.exports = (function(){
 
     var handleEntries = function(){
         var that = this;
-        return backendClient.getLiveEntriesForMediaServer().then(function(liveEntries){
+        return  adapter.getLiveEntries().then(function(liveEntries){
```

Comment:
How do we determine the type of the adapter? Right now AdaptorFactory returns only wowzaAdaptor, but in the future?